packages/clay-card/src/CardWithNavigation.tsx

```diff
@@ -74,41 +74,56 @@ export const ClayCardWithNavigation: React.FunctionComponent<IProps> = ({
 				</ClayCard.AspectRatio>
 			)}
 
-			<ClayCard.Body>
-				{!horizontal && (
-					<>
-						<ClayCard.Description displayType="title" truncate>
-							{title}
-						</ClayCard.Description>
-						<ClayCard.Description displayType="text" truncate>
-							{description}
-						</ClayCard.Description>
-					</>
-				)}
-
-				{horizontal && (
-					<ClayCard.Row>
-						<span className="autofit-col">
-							<ClaySticker inline>
-								<ClayIcon
-									spritemap={spritemap}
-									symbol={horizontalSymbol}
-								/>
-							</ClaySticker>
-						</span>
-						<span className="autofit-col autofit-col-expand">
-							<span className="autofit-section">
+			{(horizontal || title || description) && (
```

Comment:
I'm looking at this in GitHub, so this might just be @wincent's most beloved UI playing tricks, but how did this change make it into this PR? Was it always here?

Reply:
Yeah... probably just appears in the diff because we merged upstream... :sigh:

Reply:
![E5R-Rwjp_400x400](https://user-images.githubusercontent.com/7074/82432156-ecad4280-9a8f-11ea-9097-3e5cea371f11.jpg)